lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -333,6 +343,17 @@ def translate_hash_to_trigger(puppet_trigger)
       self.fail "Unknown schedule type: #{puppet_trigger["schedule"].inspect}"
     end
 
+    if puppet_trigger['minutes_interval']
+      integer_interval = Integer(puppet_trigger['minutes_interval'])
+      self.fail 'minutes_interval must be an integer greater or equal to 0' unless integer_interval >= 0
+      trigger['minutes_interval'] = integer_interval
+    end
+    if puppet_trigger['minutes_duration']
+      integer_duration = Integer(puppet_trigger['minutes_duration'])
+      self.fail 'minutes_duration must be an integer greater than minutes_interval' unless integer_duration > integer_interval || (integer_duration == 0 && integer_interval == 0)
```

Comment:
I don't see how this could ever happen. If you're not using repeat triggers, they're either nil (not filled in in the manifest) and will be set to 0 by the dummy_trigger (if anything else of the trigger changes) as it always was even without repeat triggers. The API doesn't support putting a greater interval than duration so it won't be coming from current repeat triggers either.